### PR TITLE
Cleanup and consolidation of editor scripts into a single location

### PR DESCRIPTION
**Pull request by StephenHodgson**
## Overview

- Consolidated the editor and inspector folders
- A bit of cleanup and formatting of the oculus manifest preprocessor class

### CODE REVIEW

File: XRTK.Oculus/Packages/com.xrtk.oculus/Editor/AndroidManifest.OVRSubmission.xml [hunk 1]
```diff
@@ -1,23 +0,0 @@
-<?xml version="1.0" encoding="utf-8" standalone="no"?>
```
Comment (SimonDarksideJ) [inline XRTK.Oculus/Packages/com.xrtk.oculus/Editor/AndroidManifest.OVRSubmission.xml]: Why has this file been removed?  It is critical to the Oculus platform to apply a standard Android manifest for operation on the quest.

All these files that have been deleted are critically required

Comment (StephenHodgson) [reply XRTK.Oculus/Packages/com.xrtk.oculus/Editor/AndroidManifest.OVRSubmission.xml]: It wasn't removed. it was moved into a different folder

**Comment by StephenHodgson** [comment]
Consolidating these folders makes sense as they're both only run in the editor.

I could care less about the exact name of the root folder (i.e. Inspector vs Editor) but we really don't need to have an additional csproj bc of it.

**Comment by StephenHodgson** [comment]
Not only that but moving the post build processor into a build folder made sense as well.

**Comment by StephenHodgson** [comment]
It seems as though the files being moved didn't make it into the commit check-in. I'll be sure to double check the files are still there.

File: XRTK.Oculus/Packages/com.xrtk.oculus/Editor/AndroidManifest.OVRSubmission.xml (hunk 1 above)
Comment (StephenHodgson) [reply XRTK.Oculus/Packages/com.xrtk.oculus/Editor/AndroidManifest.OVRSubmission.xml]: Ah figured it out. The new folder name was one of the ignored types. Renamed and re added these files correctly now.

File: XRTK.Oculus/Packages/com.xrtk.oculus/Inspectors/BuildTools/OculusManifestPreprocessor.cs [hunk 2]
```diff
@@ -19,76 +19,78 @@
...
         {
-            var so = ScriptableObject.CreateInstance(typeof(OculusUpdaterStub));
+            var so = ScriptableObject.CreateInstance(typeof(OculusEditorAssetLocator));
             var script = MonoScript.FromScriptableObject(so);
-            string assetPath = AssetDatabase.GetAssetPath(script);
-            string editorDir = Directory.GetParent(assetPath).FullName;
-            string srcFile = editorDir + "/AndroidManifest.OVRSubmission.xml";
+            var assetPath = AssetDatabase.GetAssetPath(script);
+            var editorDir = Directory.GetParent(assetPath).FullName;
+            var srcFile = $"{editorDir}/Build/AndroidManifest.OVRSubmission.xml";
```
Comment (SimonDarksideJ) [inline XRTK.Oculus/Packages/com.xrtk.oculus/Inspectors/BuildTools/OculusManifestPreprocessor.cs]: File path does not match the new location.

Can you test that the file is generated in the correct location and does not cause an error.

Comment (StephenHodgson) [inline XRTK.Oculus/Packages/com.xrtk.oculus/Inspectors/BuildTools/OculusManifestPreprocessor.cs]: Great catch, I forgotten about this when I renamed the folder just now. 

```suggestion
            var srcFile = $"{editorDir}/BuildTools/AndroidManifest.OVRSubmission.xml";
```

**Comment by StephenHodgson** [comment]
@SimonDarksideJ any update to this?

I addressed the issues

**Comment by SimonDarksideJ** [comment]
Changes look good, but I'll need to test on a device to confirm (to ensure the manifest is still generated properly)

**Pull request merged**